dotnet4/map.jinja

```diff
@@ -47,6 +47,11 @@
         '4.7.02558' : 'KB4033393',
         '4.7.03062' : 'KB4054590',
         '4.8.03761' : 'KB4486129',
+    },
+    '2019Server': {
+        '4.7.03190' : 'KB4534119',
+        '4.8.03761' : 'KB4486129',
+        '4.8.03761' : 'KB4486153',
```

Comment:
this key is the same as the prior one...

Reply:
The same `4.8.03761` version is also installed in the newer KB.  I wanted to leave the older KB just in case some version of WS2019 might have it installed.  If you think that is not likely, I can remove it.

Reply:
Only one key will ever resolve (should be the last one, but I'm not entirely sure how the jinja implementation handles it). You can't say, "give me the second of these identical keys." You can only specify the key itself, e.g. `dotnet_ver_to_hotfix['2019Server']['4.8.03761']`

Reply:
Got it.  I'll remove the older KB one.